src/components/collapsible-section/collapsible-section.tsx

```diff
@@ -92,6 +99,16 @@ export class CollapsibleSection {
         }
     }
 
+    private handleKeyDown(event: KeyboardEvent) {
```

Comment:
Use this syntax instead and then remove the binding in the constructor 🙂 

```suggestion
    private handleKeyDown = (event: KeyboardEvent) => {
```

Reply:
In this case I prefer to change all of the constructor bindings to this. Is it OK? Or we do it for all components in one big PR? :)

Reply:
I don't know how we should do it but in this file the "new" syntax is already used for one function `handleActionClick`. 🙂

Reply:
Ah OK then. I'll fix, or you do a fixup if you want. :)

Reply:
I can fix it :)